tools/assist.py

```diff
@@ -1,19 +1,17 @@
-#!/usr/bin/python3
+#!/usr/bin/env python3
 
-import argparse
+import argparse, os, sys
 from collections import OrderedDict
-import os
-import re
-import pickle
-import sys
+
+gAddressWidth = 18 # if your ld >= 2.40 change this to 10
 
 script_dir = os.path.dirname(os.path.realpath(__file__))
 root_dir = script_dir + "/../"
-asm_dir = root_dir + "asm/non_matchings/overlays/actors"
+asm_dir = root_dir + "asm/non_matchings/overlays"
```

Comment:
Why?

(btw I'm not even sure why this script is being updated?)

Reply:
It is because it also parses the mapfile and was broken (though not many people even use this script which is why I put pretty low effort in "fixing" it).